Assets/Scripts/KuriTextManager.cs

```diff
@@ -52,6 +57,29 @@ public enum PRIORITY {
             audioSource.volume = 0.05f;
         }
 
+        public void ToggleGlow()
+        { //toggles glow on and off by changing material, helper method for chest lights
+            if (isGlowing)
+            {
+                for(int i = 0; i < lights.Length; i++)
```

Comment:
use a foreach loop: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/foreach-in